flink/src/main/java/org/apache/iceberg/flink/source/RowDataFileReader.java

```diff
@@ -47,51 +45,51 @@
 import org.apache.iceberg.types.TypeUtil;
 import org.apache.iceberg.util.PartitionUtil;
 
-class RowDataIterator extends DataIterator<RowData> {
+public class RowDataFileReader implements FileReader<RowData> {
```

Comment:
Nit: Better to add a comment indicating that the RowDataFileReader is designed for scanning rows from FileScanTask. Because as a reviewer, I cannot see the meaning directly from the class naming ( I think it's a historical issue in the initial review :-) )

Reply:
there is Javadoc in the interface `FileScanTaskReader`. I think that is the right place for the Javadoc.